x-pack/plugins/actions/server/action_type_registry.test.ts

```diff
@@ -31,6 +31,7 @@ describe('register()', () => {
     actionTypeRegistry.register({
       id: 'my-action-type',
       name: 'My action type',
+      minimumLicenseRequired: 'basic',
```

Comment:
maybe there's an hook later, but this seems to violate "non-built-in action types can only set a gold+ license"

Reply:
You can set anything here but it will be validated when you register. The registration validation exists here: https://github.com/elastic/kibana/pull/58668/files#diff-adc6606fd00ebc8b46eecefae314f1b5R187 and the built ins don't go through there, they register directly to the `actionTypeRegistry`.